packages/xeus/src/index.ts

```diff
@@ -1,5 +1,3 @@
-export * from './coincident.worker';
```

Comment:
Curious to know more about the issue you are having?

For reference `jupyterlite-pyodide-kernel` still exports them fine it seems? 

https://github.com/jupyterlite/pyodide-kernel/blob/f921132f8186f6d061ae1171270c4e4d77562e59/packages/pyodide-kernel/src/index.ts#L5-L6

Reply:
As soon as you use jupyterlite-xeus as a library using something else than webpack (happens with esbuild), there are clashes. This is because those exported files go through the webpack machinery.

Reply:
I'm reverting this change in https://github.com/jupyterlite/xeus/pull/228 and providing a new webpack-free package that's more reusable, @jupyterlite/xeus will continue exporting the comlink and coincident workers.